Mathlib/CategoryTheory/Galois/Examples.lean

```diff
@@ -96,6 +96,10 @@ noncomputable instance : FiberFunctor (Action.forget FintypeCat (MonCat.of G)) w
   preservesQuotientsByFiniteGroups _ _ _ := inferInstance
   reflectsIsos := ⟨fun f (h : IsIso f.hom) => inferInstance⟩
 
+/-- The forgetful functor from finite `G`-sets to sets is a `FiberFunctor`. -/
+noncomputable instance : FiberFunctor (forget₂ (Action FintypeCat (MonCat.of G)) FintypeCat) :=
+  inferInstanceAs <| FiberFunctor (Action.forget FintypeCat (MonCat.of G))
```

Comment:
It seems that previously in this file you've used `Action.forget`, and now you are using `forget₂`. Is there a reason for using both? Or can the previous instances in this file be rewritten to only use `forget₂`?

Reply:
Things about the `Action` category tend to be stated for `Action.forget` since it is more general (it does not assume the base category to be a `ConcreteCategory`), while `forget₂` needs the base category to be concrete. In this case, `FintypeCat` is concrete, so also `forget₂` can be used (and I think that every statement about `Action.forget` should be mirrored for `forget₂` in other places of mathlib too, but currently that is not the case, so using `Action.forget` in proofs is working more smoothly).